core/indexer/elasticsearch.go

```diff
@@ -104,16 +104,16 @@ func (ei *elasticIndexer) SaveBlock(
 	}
 
 	txsSizeInBytes := computeSizeOfTxs(ei.marshalizer, txPool)
-	go ei.database.SaveHeader(headerHandler, signersIndexes, body, notarizedHeadersHashes, txsSizeInBytes)
+	ei.database.SaveHeader(headerHandler, signersIndexes, body, notarizedHeadersHashes, txsSizeInBytes)
```

Comment:
may be leave to go routines - add  a go routine throttler ?

Reply:
for the moment i will let as it is because method SaveBlock is called on a goroutine and i want the writing of header, miniblocks and transaction to be serial because this write do a lot of requests to elasticsearch database because have to check if miniblocks and transactions exits in database.